example_app/Gemfile

```diff
@@ -6,3 +6,7 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
 gem 'packer', path: '../'
 gem 'sinatra'
+
+group :development do
+  gem 'byebug'
```

Comment:
Should this be in the gemspec instead?

Reply:
👍 And maybe `sinatra` too?

Reply:
Haven't used it in the context of the gem. But should we want to use it in the future we can always add it.

Reply:
Actually, why is `sinatra` in the Gemfile?

Reply:
This is an example app to test/showcase the gem integration with Sinatra. Ideally, we should have a proper test suite with both Rails and Sinatra apps.
This is my understanding but maybe @lesleh can give more context around this.

Reply:
👍 Makes sense. And yes, we should probably have rails and sinatra test suites.